src/components/directory-content.jsx

```diff
@@ -427,18 +428,22 @@ const DirectoryContent = () => {
         );
     };
 
-    const renderEmptyDirContent = () => {
-        return (
-            <>
-                <div style={{ textAlign: 'center', marginTop: '100px' }}>
-                    <FolderOpenRoundedIcon style={{ width: '100px', height: '100px' }} />
-                    <h1>
-                        <FormattedMessage id={'emptyDir'} />
-                    </h1>
-                </div>
-            </>
-        );
-    };
+    const renderEmptyDirContent = () => (
+        <EmptyDirectory
+            openDialog={(mouseEvent) => {
```

Comment:
you can create a function handleOpenDialog

Reply:
Done.